src/main/java/core/basesyntax/Car.java

```diff
@@ -1,5 +1,62 @@
 package core.basesyntax;
 
-public class Car {
+public class Car implements Cloneable {
+    private String brand;
+    private String model;
+    private Engine engine;
+    private int weight;
+    private int manufacturedYear;
 
+    public Car(String brand, String model, Engine engine, int weight, int manufacturedYear) {
+        this.brand = brand;
+        this.model = model;
+        this.engine = engine;
+        this.weight = weight;
+        this.manufacturedYear = manufacturedYear;
+    }
+
+    @Override
+    protected Car clone() throws CloneNotSupportedException {
```

Comment:
so that it could be called from outside of this class
```suggestion
    public Car clone() throws CloneNotSupportedException {
```